web-client/src/pages/signup.js

```diff
@@ -0,0 +1,65 @@
+import '../components/signup.css';
+import '../global.css';
+
+export default function Signup(){
+    return (
+        <>
+        <header className="site-header">
```

Comment:
Since the header is common to all pages, we should move it to App.js (this is the component that defines the layout for all pages)

Reply:
Theres probably a header there already but its very simple. You should replace that one with this one here.

Reply:
Ok will do :)